packages/hawtio/src/plugins/auth/oidc/oidc-service.ts

```diff
@@ -195,9 +214,11 @@ export class OidcService implements IOidcService {
       authorizationUrl.searchParams.set('nonce', nonce)
       // authorizationUrl.searchParams.set('login_hint', 'hawtio-viewer@fuseqe.onmicrosoft.com')
       // authorizationUrl.searchParams.set('hsu', '1')
-      if (config.prompt) {
-        authorizationUrl.searchParams.set('prompt', config.prompt)
-      }
+      // it's better to stick to default, as 'prompt=login' always
+      // shows login screen...
+      // if (config.prompt) {
+      //   authorizationUrl.searchParams.set('prompt', config.prompt)
+      // }
```

Comment:
too many comment outs here and there. maybe we can clean them up now.

Reply:
I've removed the comment ;) Thanks for spotting!